### ISSUE TITLE
Deploy the model zoo to the web

### ISSUE THREAD

**Issue by logankilpatrick** [FluxML/model-zoo#312]
It seems like hosting these model zoo examples on the FluxML website would be very helpful from a readability point of view. @lilianabs any interest in this? I could definitely help as well

**Comment by logankilpatrick** [comment]
This repo would then just become the backend for that part of the site, not sure how easy this would be.

**Comment by lilianabs** [comment]
@logankilpatrick when I participated in Season of Docs, we wanted to bring all of the model zoo contents to the FluxML site but didn't manage to complete it. I love the idea but I'm not very sure how we could use this repo as a backend for the FluxM site. Maybe we can start by creating a Colab notebook for each model and publishing the full list on the FluxML site? Then we could think of automating the process? I'm definitely up for working on this as well as the contributor's guide :)

**Comment by darsnack** [comment]
Probably the easiest option is to use Publish.jl, Documenter.jl, or Pollen.jl to host a "docs" site that is really all the tutorials. And spend a bunch of time on the CSS to make it look more like a polished site than plain docs. Then just have a header button on the main website that links to this tutorial website.

The more complex option would be to switch to Franklin.jl for the main website repo, then have a function that checks out this repo and builds it all into a single tutorial page. Similar to how the blog posts page on the Julia website is autogenerated.

**Comment by DhairyaLGandhi** [comment]
Yeah, we can complete the project and have nicer tutorials rather than individual posts. Let's talk about this @lilianabs and set up a plan.

**Comment by lilianabs** [comment]
Sure! Let's talk about this @DhairyaLGandhi.
